src/components/Slideover/Slideover.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <transition name="fade" @after-enter="openSlideover = true">
```

Comment:
I believe it's good to move these expressions that shows and hides the blockUi and the slideover to methods, like `hideSlideOver`, `showSlideOver`. And, these methods should be emit the `show` and `hide` events.

Reply:
No methods have been added to **open** and **close** the transitions because when using methods we would need to change the css via js because with the methods the transitions end are asynchronous, and as we use scss files the methods do not apply.